test/core/vm/actor/builtin/v2/power/storage_power_actor_v2_test.cpp

```diff
@@ -192,7 +192,7 @@ namespace fc::vm::actor::builtin::v2::storage_power {
     constructed();
     callerCodeIdIs(kEmptyObjectCid);
 
-    EXPECT_OUTCOME_ERROR(VMExitCode::kSysErrForbidden,
+    EXPECT_OUTCOME_ERROR(ABORT_CAST(VMExitCode::kSysErrForbidden),
```

Comment:
Maybe add "EXPECT_ABORT(exit, expr)" macro for actors tests?

Reply:
Maybe later if it needed.
@Alexey-N-Chernyshov, as I remember you want to remake actor tests, you could take this idea